index.js

```diff
@@ -0,0 +1,5 @@
+function welcomeAlert(){
+alert("Welcome to a new journey and my new site!");
```

Comment:
code looks great! one thing you can do is in code blocks (anythign between teh two {}) you can indent two spaces to improve readability.
```
function welcomeAlert(){
  alert("Welcome to a new journey and my new site!");

Reply:
Thanks, I typically forget even thought I downloaded prettier. Again, thanks for remembering to check on us!(: